Tools/CISupport/ews-build/config.json

```diff
@@ -489,38 +449,22 @@
       "type": "Periodic", "name": "safe-merge-queue", "periodicBuildTimer": 900,
       "builderNames": ["Safe-Merge-Queue"]
     },
-    {
-      "type": "Triggerable", "name": "macos-monterey-release-build-ews",
-      "builderNames": ["macOS-Monterey-Release-Build-EWS"]
-    },
     {
       "type": "Triggerable", "name": "macos-ventura-release-build-ews",
       "builderNames": ["macOS-Ventura-Release-Build-EWS"]
     },
     {
-      "type": "Triggerable", "name": "macos-monterey-release-wk1-tests-ews",
-      "builderNames": ["macOS-Monterey-Release-WK1-Tests-EWS"]
+      "type": "Triggerable", "name": "macos-ventura-release-wk1-tests-ews",
+      "builderNames": ["macOS-Ventura-Release-WK1-Tests-EWS"]
     },
     {
-      "type": "Triggerable", "name": "macos-applesilicon-ventura-release-wk1-tests-ews",
-      "builderNames": ["macOS-AppleSilicon-Ventura-Release-WK1-Tests-EWS"]
-    },
-    {
-      "type": "Triggerable", "name": "macos-monterey-release-wk2-tests-ews",
-      "builderNames": ["macOS-Monterey-Release-WK2-Tests-EWS"]
-    },
-    {
-      "type": "Triggerable", "name": "macos-applesilicon-ventura-release-wk2-tests-ews",
-      "builderNames": ["macOS-AppleSilicon-Ventura-Release-WK2-Tests-EWS"]
+      "type": "Triggerable", "name": "macos-ventura-release-wk2-tests-ews",
+      "builderNames": ["macOS-Ventura-Release-WK2-Tests-EWS"]
     },
     {
       "type": "Triggerable", "name": "macos-release-wk2-stress-tests-ews",
       "builderNames": ["macOS-Release-WK2-Stress-Tests-EWS"]
     },
-    {
-      "type": "Triggerable", "name": "macos-applesilicon-ventura-release-wk2-stress-tests-ews",
-      "builderNames": ["macOS-AppleSilicon-Ventura-Release-WK2-Stress-Tests-EWS"]
-    },
     {
       "type": "Triggerable", "name": "macos-applesilicon-sonoma-debug-build-ews",
       "builderNames": ["macOS-AppleSilicon-Sonoma-Debug-Build-EWS"]
```

Comment:
Do we want to remove `AppleSilicon` from Sonoma queue names as well for consistency?

Reply:
I echo this same sentiment. We should remove the AppleSilicon for consistency. Otherwise, I'd say this change is ready to land.